packages/jest-environment/src/index.ts

```diff
@@ -44,6 +44,11 @@ export declare class JestEnvironment {
   fakeTimersLolex: LolexFakeTimers | null;
   moduleMocker: jestMock.ModuleMocker | null;
   runScript<T = unknown>(script: Script): T | null;
+  compileFunction?<T = unknown>(
```

Comment:
I've made it optional so we're not blocked by jsdom having support. It'll silently use it if it's ever implemented